src/ensembl/src/content/app/entity-viewer/state/general/entityViewerGeneralSelectors.ts

```diff
@@ -23,19 +23,19 @@ import { EnsObjectGene } from 'src/shared/state/ens-object/ensObjectTypes';
 export const getEntityViewerActiveGenomeId = (state: RootState) =>
   state.entityViewer.general.activeGenomeId;
 
-export const getEntityViewerActiveEnsObjectIds = (state: RootState) =>
-  state.entityViewer.general.activeEnsObjectIds;
+export const getEntityViewerActiveEntityIds = (state: RootState) =>
+  state.entityViewer.general.activeEntityIds;
 
-export const getEntityViewerActiveEnsObjectId = (state: RootState) => {
-  const activeEnsObjectIds = getEntityViewerActiveEnsObjectIds(state);
+export const getEntityViewerActiveEntityId = (state: RootState) => {
+  const activeEntityIds = getEntityViewerActiveEntityIds(state);
   const activeGenomeId = getEntityViewerActiveGenomeId(state);
-  return activeGenomeId ? activeEnsObjectIds[activeGenomeId] : null;
+  return activeGenomeId ? activeEntityIds[activeGenomeId] : null;
 };
 
-export const getEntityViewerActiveEnsObject = (
+export const getEntityViewerActiveEntity = (
   state: RootState
 ): EnsObjectGene | null => {
-  const activeObjectId = getEntityViewerActiveEnsObjectId(state);
+  const activeObjectId = getEntityViewerActiveEntityId(state);
   if (!activeObjectId) {
     return null;
   }
```

Comment:
The line below says `return getEnsObjectById(state, activeObjectId) as EnsObjectGene;`

I can't quite remember why we were doing this, so am a bit confused here. Do we still need this dependency on the ensObjects part of the redux store here?

Reply:
It looks like we don't need this anymore. I've removed it